src/ReactiveDomain.Foundation/StreamStore/JsonMessageSerializerSettings.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Reflection;
+
+// ReSharper disable once CheckNamespace
+namespace ReactiveDomain.Foundation {
+    public class JsonMessageSerializerSettings
+    {
+        public bool ThrowOnTypeNotFound;
+        public Assembly AssemblyOverride;
+        public bool FullyQualify;
+        /// <summary>
+        /// Create a new settings object for the JsonMessageSerializer
+        /// </summary>
+        /// <param name="fullyQualify">Instructs the Serializer to write the fully qualified Assembly name and version. This can cause types and not to be found if with assembly version changes even if the type has not changed.</param>
+        /// <param name="assemblyOverride">Replaces the recorded assembly when deserializing</param>
+        /// <param name="throwOnTypeNotFound">Will throw and Invalid Operation Exception if a target type cannot be found, rather than returning a JObject</param>
```

Comment:
```suggestion
        /// <param name="throwOnTypeNotFound">Will throw an Invalid Operation Exception if a target type cannot be found, rather than returning a JObject</param>
```